File: LeetCode/Easy/1-Two-Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> sum2;
        for(int i=0;i<nums.size();i++){
            for(int j=1;j<nums.size();j++){
                if(nums[i]+nums[j] == target){
                    sum2.push_back(i);
                    sum2.push_back(j);
                }
            }
        }
        return sum2;
    }
};

File: LeetCode/Easy/1-Two-Sum.java
import java.util.*;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int arr[] = new int[2];
        Map<Integer, Integer> map = new HashMap<> ();
        for(int i=0; i<nums.length; i++){
            if(map.containsKey(target-nums[i]) == true){
                arr[0] = i;
                arr[1] = map.get(target-nums[i]);
            }else{
                map.put(nums[i], i);
            
            }
        }
        return arr;
    }
}